app/src/ui/app.tsx

```diff
@@ -746,12 +746,15 @@ export class App extends React.Component<IAppProps, IAppState> {
         repository,
       })
     } else {
-      this.props.dispatcher.removeRepositories([repository])
+      this.props.dispatcher.removeRepositories([repository], false)
```

Comment:
I was worried that, for cases where the user has said they don't want to be prompted, the app will use the set value for `deleteRepoFromDisk` and you'd potentially find yourself in a situation where you weren't aware you were moving repositories to the trash. But this seems like a good default if you choose not to be prompted about removing it.

Reply:
Yea it kinda sucks to me because I think there might need to be another setting for this. Just cause when you are avoiding prompts you may still want to delete them from disk? just thought it would be better than it was today to allow the option when prompted.

Reply:
I'm inclined to put this aside for now and see if we get any reports on remembering this setting across sessions. I think we will, and I think we can track this setting without need to have a UI for it. But let's not overthink this change just yet.